contents/docs/getting-started/group-analytics.mdx

```diff
@@ -8,25 +8,28 @@ import CreateGroupType from "./\_snippets/group-type-create.mdx"
 import ChatGroupType from "./\_snippets/group-type-create-chat.mdx"
 import SetGroupProperties from "./\_snippets/set-group-properties.mdx"
 
-> **Note: ** This is a paid feature and is not available on the open-source or free cloud plan. Learn more [here](/pricing).
+> **Note: ** This is a paid feature and is not available on the open source or free cloud plan. See our [pricing page](/pricing) for more.
 
-Group analytics is a powerful feature in PostHog that allows you to perform analytics on entities other than single users.
-This feature is slightly more advanced, but opens up entirely new avenues for analytics which are essential for most B2B use cases.
+Group analytics is a powerful feature in PostHog that allows you to perform analytics on entities (e.g. companies, teams etc.) other than single users.
+This feature is slightly more advanced, but opens up entirely new avenues for analytics that are essential for most B2B use cases.
 
-Group analytics can sometimes feel confusing to new users, and this often times comes from skipping over the basic concepts too quickly.
+Group analytics can sometimes feel confusing to new users, and this often comes from skipping over the basic concepts too quickly.
 We'll start with an overview of how Group analytics works, and then jump into creating new groups and viewing them within PostHog.
 
 ## 1. What is a group?
 
 In PostHog, a 'Group' is an entity that we can send events and hold properties that is beyond a single user.
 
-To get a more concrete idea of what this looks like in practice, here are a couple examples of common entities that we can model using groups:
+Here are two examples of common entities we can model using groups:
 
-1. **Organizations/Teams**: It's very common in B2B apps to have multiple users under the same 'Team' account, in which case groups allow us to perform analytics on individual teams exactly as if they were users.
-2. **Threads/Chats/Posts**: If you're creating a social app, you might want to track events that are connected with specific conversations such as replies
+1. **Organizations/Teams**: It's very common in B2B apps to have multiple users under the same 'Team' account, in which case groups allow us to perform analytics on individual teams as if they were users.
+
+2. **Threads/Chats/Posts**: If you're creating a social app, you might want to track events that are connected with specific conversations, such as replies.
 
 Looking at these examples, it might be tempting to think that a group is just a group of users; however, this is not exactly the case.
+
 We'll see this later, but the basic idea is that **only events are connected to groups directly, and any user who sent one of those events will also be linked indirectly**.
+
 In this way, there isn't a way to directly 'assign' a user to a group - instead, this user would send an event with a special property linking the event to a group.
```

Comment:
Interested in your input here @Lior539 / @ivanagas.

Perhaps it's my lack of technical background, but this explanation of groups is quite confusing to me. Is there a better way to explain this?

Reply:
I dont know what this sentence means tbh.

It could be worth calling out cohorts here again e.g.

`there isn't a way to directly 'assign' a user to a group – but you should use cohorts for that` (but phrase it better)